docs(operator): document the continuous scanning capability (#25)
kubescape/kubescape.io: docs/docs/operator/continuous-scanning.md

## Installation

Continuous Scanning is built into the Kubescape Operator Helm chart. To use this capibility, you only need to enable it. Start by navigating to the `values.yaml` file and make sure that the corresponding `capabilities.continuousScan` key is set to `enabled`, like so:

```yaml
capabilities:
  continuousScan: enable  # Make sure this is set to "enable"
```

Once you apply the chart with the capability enabled, Kubescape will continuously secure your cluster and provide the scan results as Custom Resources.

## Accessing Results

### For the Whole Cluster

Kubescape provides scan results as Custom Resources so you can access them in the same convenient way you access other Kubernetes objects. Let’s assume you’d like to see a birds-eye view of you cluster’s security. In Kubescape terms, that would mean taking a look at the cluster-level configuration scan summary:

```
kubectl get workloadconfigurationscansummaries -o yaml
```

> Running this command will return you a YAML-formatted list of configuration scan summaries for your cluster by namespaces.

Reviewer: ```suggestion
Running this command will return a YAML-formatted list of configuration scan summaries for your cluster by namespaces.
```

Author: Fixed.

> On clusters with many namespaces, the results might be overwhelming and might even exceed your terminal history. Since Kubescape serves results as Kubernetes objects, which are YAML files at its core, you can do your usual tricks: pipe them to files, text editors etc. A trick we commonly use is:

Reviewer: ```suggestion
On clusters with many namespaces, the results might be overwhelming and might even exceed your terminal history. Since Kubescape serves results as Kubernetes objects, which are YAML files at the end of the day, you can apply your usual processes to aggregate them in a readable way. For example - pipe them to files, text editors etc. We commonly use the following command:
```

Author: Fixed.

```
kubectl get workloadconfigurationscansummaries -o yaml | less
```

> This way you get the entire results and browse the file as you see fit.

Reviewer: ```suggestion
This way you get the complete result in a file and can browse it file as you see fit.
```

Author: Fixed.

### By Namespace

Let’s say you have a namespace `k8s-bad-practices`. It runs badly misconfigured insecure workloads and you would like to see how Kubescape sees them. To get the configuration scan results for this namespace, run the following command:

```
kubectl get -n k8s-bad-practices workloadconfigurationscansummaries -o yaml | less
```

You should see a summary for the insecure namespace only.

### By Workload

> You could also be interested in checking how secure a specific workload is. To see the results, use:

Reviewer: ```suggestion
You could also be interested in checking how secure a specific workload is. To see those results, use:
```

Author: Fixed.

```
kubectl get -n k8s-bad-practices workloadconfigurationscansummaries trusty-reverse-proxy -o yaml | less
```

That should provide you with a configuration scan summary for this workload.

## How It Works

> With Continuous Scanning enabled, Kubescape continuously monitors a cluster for changes, determines how they affect the security of your cluster and reflect their security impact in the overall cluster’s security report, as well as individual workload security reports. Let’s take a look at a sequence diagram.

Reviewer: ```suggestion
With Continuous Scanning enabled, Kubescape continuously monitors a cluster for changes, determines how they affect the security of your cluster and reflects their security impact in the overall cluster’s security report, as well as individual workload security reports. Let’s take a look at a sequence diagram.
```

Author: Fixed.

> ```mermaid

Reviewer 2: This very lovely diagram is the kind of thing that should be on a Markdown page in GitHub as opposed to the user documentation, in my opinion.

Author: I agree with you. While Ben and David mentioned that the idea was to have all documentation in one place, this diagram is already [present](https://github.com/kubescape/helm-charts/blob/4bf7b6715992d83d1f72a66af33a6019573f4924/charts/kubescape-operator/README.md#how-it-works) in the Helm chart. Moreover, the kubescape.io docs seem to be more of a How-To Guide instead of a How-It-Works Guide, making this diagram seem out of place. I will remove it from here and leave only the portion in the Helm chart.

Author: I removed the diagram and the corresponding extension.
